Modules/Filtering/BiasCorrection/include/itkN4BiasFieldCorrectionImageFilter.hxx

```diff
@@ -66,11 +67,49 @@ N4BiasFieldCorrectionImageFilter<TInputImage, TMaskImage, TOutputImage>
   this->m_MaximumNumberOfIterations.Fill( 50 );
 }
 
+
+template<typename TInputImage, typename TMaskImage, typename TOutputImage>
+void
+N4BiasFieldCorrectionImageFilter<TInputImage, TMaskImage, TOutputImage>
+::VerifyPreconditions() ITKv5_CONST
+{
+  Superclass::VerifyPreconditions();
+
+  const InputImageType *const inputImage = this->GetInput();
+
+  if (inputImage == nullptr)
```

Comment:
This check should not be needed. This should be the "PrimaryInput" according to the ProcessObject, and it should by default be marked as required. This should be verified in the base class: https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Core/Common/src/itkProcessObject.cxx#L1419-L1429

Reply:
@blowekamp  Thanks for your feedback. I'll remove `itkExceptionMacro("Failed precondition: Input image should not be null")` with the next amend.